lib/features/thread/data/network/thread_isolate_worker.dart

```diff
@@ -66,12 +66,13 @@ class ThreadIsolateWorker {
     EmptyMailboxFolderArguments args,
     TypeSendPort sendPort
   ) async {
-    final rootIsolateToken = args.isolateToken;
-    BackgroundIsolateBinaryMessenger.ensureInitialized(rootIsolateToken);
-    await HiveCacheConfig.instance.setUp();
-
-    List<EmailId> emailListCompleted = List.empty(growable: true);
     try {
+      final rootIsolateToken = args.isolateToken;
+      BackgroundIsolateBinaryMessenger.ensureInitialized(rootIsolateToken);
```

Comment:
what is this purpose?

Reply:
From flutter 3.7 to use plugins and platform channels from any isolate we need BackgroundIsolateBinaryMessenger. This is not a new addition, please read more from previous issues.